qiskit/circuit/quantumcircuit.py

```diff
@@ -1858,10 +1858,10 @@ def draw(
                 Default is True, except for when ``output`` is set to  ``"text"``.
             wire_order (list): Optional. A list of integers used to reorder the display
                 of the bits. The list must have an entry for every bit with the bits
-                in the range 0 to (num_qubits + num_clbits).
+                in the range 0 to (``num_qubits`` + ``num_clbits``).
 
         Returns:
-            :class:`TextDrawing` or :class:`matplotlib.figure` or :class:`PIL.Image` or
+            :class:`.TextDrawing` or :class:`matplotlib.figure` or :class:`PIL.Image` or
             :class:`str`:
```

Comment:
You probably already know this, but the `:class:` directive won't really create a link for `matplotlib.figure`, `PIL.Image` and `str`, because these are not classes documented in the scope of`qiskit-terra`. It will just have the same effect as writing the text between backticks. Therefore... wouldn't it be clearer to use the backticks directly? (also, shouldn't `matplotlib.figure` be `matplotlib.figure.Figure` ?)

```suggestion
            :class:`.TextDrawing` or ``matplotlib.figure.Figure`` or ``PIL.Image`` or
            ``str``:
```

FYI, you can add links to external documentation by:

1. If they use sphinx to build their docs, you can use [intersphinx_mapping](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html)
2. Manually [adding a link](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-custom-link-text) to whatever URLs are of interest.

Reply:
This is all correct, yeah.  Matplotlib do use Sphinx - on other Qiskit projects we actually do intersphinx to it, so I'm a bit surprised we aren't in Terra already.

Reply:
ill push for intersphinx in the metapackage next week